src/utils/api.js

```diff
@@ -43,10 +52,198 @@ export const getClusters = (
         swlat,
         nelat,
         zoom,
-        muni,
+        muni: muni ? 1 : 0,
+        t: types,
+      },
+    }),
+  )
+
+export const getLocations = (
+  { swlng, nelng, swlat, nelat },
+  includeMunicipal = false,
+  categories = null,
+  types = null,
+  locale = LOCALE.EN,
+  invasive = false,
+  limit = 1000,
+  offset = 0,
+  reviews = false,
+  { lng = 0, lat = 0 },
+) =>
+  handleResponse(
+    instance.get('/locations.json', {
+      params: {
+        swlng,
+        nelng,
+        swlat,
+        nelat,
+        municipal: includeMunicipal ? 1 : 0,
+        c: categories,
+        t: types,
+        invasive: invasive ? 1 : 0,
+        limit,
+        offset,
+        locale,
+        reviews: reviews ? 1 : 0,
+        lng,
+        lat,
+      },
+    }),
+  )
+
+export const postLocations = (
+  { lng, lat },
+  type_ids,
+  author = null,
+  description = null,
+  season_start = 0,
+  season_stop = 0,
+  no_season = false,
+  unverified = false,
+  access = 0,
+  comment = null,
+  fruiting = 0,
+  quality_rating = 0,
+  yield_rating = 0,
+  observed_on = null,
+  photo_file_name = null,
+  photo_data,
+) =>
+  handleResponse(
+    instance.post('/locations.json', photo_data, {
+      params: {
+        lng,
+        lat,
+        type_ids,
+        author,
+        description,
+        season_start,
+        season_stop,
+        no_season,
+        unverified,
+        access,
+        comment,
+        fruiting,
+        quality_rating,
+        yield_rating,
+        observed_on,
+        photo_file_name,
+      },
+    }),
+  )
+
+export const getLocationById = (id, locale = LOCALE.en) =>
+  handleResponse(
+    instance.get('/locations/{id}.json', {
+      params: {
+        id,
+        locale,
+      },
+    }),
+  )
+
+export const editLocation = (
+  id,
+  { lng, lat },
+  type_ids,
+  author = null,
+  description = null,
+  season_start = 0,
+  season_stop = 0,
+  no_season = false,
+  unverified = false,
+  access = 0,
+) =>
+  handleResponse(
+    instance.post('/locations/{id}.json', null, {
+      params: {
+        lng,
+        lat,
+        type_ids,
+        author,
+        description,
+        season_start,
+        season_stop,
+        no_season,
+        unverified,
+        access,
+      },
+    }),
+  )
+
+export const getTypes = (
+  swlng = null,
+  nelng = null,
+  swlat = null,
+  nelat = null,
+  zoom = 0,
+  muni = false,
+  locale = LOCALE.EN,
+  types = null,
+  urls = false,
+  categories = null,
+  uncategorized = false,
+  pending = false,
+) =>
+  handleResponse(
+    instance.get('/types.json', {
+      params: {
+        swlng,
+        nelng,
+        swlat,
+        nelat,
+        zoom,
+        muni: muni ? 1 : 0,
+        locale,
         t: types,
+        urls: urls ? 1 : 0,
+        categories,
+        uncategorized: uncategorized ? 1 : 0,
+        pending: pending ? 0 : 1,
```

Comment:
Is there a particular reason this is 0 : 1 and not the other way around?

Reply:
That was an accident, but I fixed it. Thanks! :)